tests/ceph_volume/test_ceph_volume_get_inventory_list.py

```diff
@@ -0,0 +1,41 @@
+from cli.exceptions import OperationFailedError
+
+
+def run(ceph_cluster, **kw):
+    """
+    Verify ceph-ansible fails to get an inventory
+    list with KeyError: 'ceph.cluster_name' in json format
+    but not in plain format
+    """
+
+    installer = ceph_cluster.get_nodes(role="installer")[0]
+
+    # Non bootstrap osd Node
+    non_bs_node = list(set(ceph_cluster.get_nodes(role="osd")) - {installer})[0]
+
+    # Note: This cmd uses rhceph-5-rhel8:latest, the requirement is just to ensure whether
+    # the ceph inventory is able to list the details and any container image is sufficient
+    cmd = (
+        "podman run --rm --privileged --net=host --ipc=host --ulimit nofile=1024:4096 -v "
```

Comment:
@pranavprakash20 can we use run method [1] here?

[1]  https://github.com/red-hat-storage/cephci/blob/master/cli/utilities/containers.py#L54

Reply:
+1